lib/Net/Google/Analytics/Request.pm

```diff
@@ -43,7 +44,9 @@ sub _params {
 
     my @required = qw(ids metrics);
 
-    if (!$self->{realtime}) {
+	if ($self->{realtime}) {
+		$self->{service} = 'realtime';
+	} else {
         push(@required, qw(start_date end_date));
```

Comment:
This happens too late (`_param` is called after `_uri`) and causes a test failure.